tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1378,6 +1378,7 @@ def test_sum_invalid(is_module, xp, shape, axis, keepdims, dtype):
         'input': ['random'],
         'scalar_value': [1],
         'is_scalar_rhs': [False],
+        'dodge_nondifferentiable': [True],
```

Comment:
How about adding this as a class atribute?
It should be safe to enable it in other test configurations as well.